telegram/_bot.py

```diff
@@ -6339,6 +6338,9 @@ async def create_new_sticker_set(
             Removed the deprecated parameters mentioned above and adjusted the order of the
             parameters.
 
+        .. versionremoved:: NEXT.VERSION
```

Comment:
~is versionremoved a thing? You mean versionchanged?~

TIL: https://github.com/sphinx-doc/sphinx/issues/11480

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/pull/4217/commits/23f5361c6d43604b06392a15512e10f625af2b1c 😘